roles/default/kiali-deploy/templates/kubernetes/role-viewer.yaml

```diff
@@ -47,10 +47,7 @@ rules:
   - list
   - watch
 - apiGroups:
-  - config.istio.io
   - networking.istio.io
-  - authentication.istio.io
-  - rbac.istio.io
```

Comment:
I think you also want to do this to the openshift role-viewer.yaml template.  This: https://github.com/kiali/kiali-operator/blob/master/roles/default/kiali-deploy/templates/openshift/role-viewer.yaml#L49-L54